service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocumentDownloadService.java

```diff
@@ -23,33 +21,56 @@
 
 @Slf4j
 @Service
-@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class DocumentDownloadService {
     private final AuthTokenGenerator authTokenGenerator;
     private final DocumentDownloadClientApi documentDownloadClient;
     private final IdamClient idamClient;
     private final RequestData requestData;
 
+    private final SecureDocStoreService secureDocStoreService;
+    private final FeatureToggleService featureToggleService;
+
+    public DocumentDownloadService(AuthTokenGenerator authTokenGenerator,
```

Comment:
Look at whether we need this as part of checking the component scan/beans stuff.

Reply:
looks like we don't need this change, and can keep the original one